interceptor/pkg/interceptor/pdinterceptor.go

```
@@ -58,11 +57,16 @@ type Organization struct {
 }
 
 type interceptorHandler struct {
-	PDToken string
+	PDToken      string
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)